src/Model/Package.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+use BringYourOwnIdeas\Maintenance\Tasks\UpdatePackageInfo;
+
+/**
+ * Describes an installed composer package version.
+ */
+class Package extends DataObject
+{
+    private static $db = [
+        'Name' => 'Varchar(255)',
+        'Description' => 'Varchar(255)',
+        'Version' => 'Varchar(255)',
+        "Type" => 'Varchar(255)',
+    ];
+
+    private static $summary_fields = [
+        'Title',
+        'Description',
+        'Version',
+    ];
+
+    public function getTitle()
+    {
+        $niceName = preg_replace('#^[^/]+/(silverstripe-)?#', '', $this->Name);
+        $niceName = explode('-', $niceName);
+        $niceName = implode(' ', $niceName);
+        $niceName = ucwords($niceName);
+        return $niceName;
+    }
+
+    public function getSummary()
+    {
+        return $this->renderWith('Package_summary');
```

Comment:
Out of interest, assuming this has HTML in it, how does this look when exporting to CSV?

Reply:
It looks like a string of HTML in a cell ;)
Thus it isn't included in the default `summary_fields` :)